errors/errno.go

```diff
@@ -2,18 +2,16 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package linux
+package errors
 
-import (
-	"fmt"
-)
+import "fmt"
 
-// Errno is a Linux error number on every GOOS.
+// Errno is a 9P2000.L/Linux error number on every GOOS.
```

Comment:
Something not touched on here -- but you can see it in a comment below -- the error numbers defined here are for amd64. I don't know if 9P2000.L ever thought about client and server architecture not being the same (and I'm fine taking the PR as is, since this is an exising deficiency...), but it occurs to me that clients actually have no established way to tell the server "this is my architecture" or vice versa.

Reply:
Yeah I was concerned about this as well.
I'll remove the addition to the comment and we can punt on cross arch until it's a problem.

An approach that might work in the future is using invented pkg values like these, and then doing translation as early/late as possible at the edges.
So something like `linux.E*` gets translates to `syscall.E*` on Linux systems by the server and client.
For non-POSIX system they could either remain as-is or maybe also get translated.

In any case, yeah - we don't have to think about this right now.

Reply:
I think we'd just have to have definitions of each value per platform, under the assumption that the client is the same architecture as the server.

Alternatively, define all the values in a large map[string] indexed by runtime.GOARCH by default, but the user can provide an alternative value in the p9.Attacher when starting the server by configuration.